azure-pipelines.yml

```diff
@@ -0,0 +1,84 @@
+trigger:
+  branches:
+    include:
+      - master
```

Comment:
If my understanding is correct - this will trigger a build only for the `master` branch. Perhaps at some point we should open this for all branches.

Reply:
sure! Maybe we can have prefix for these branches